tests/screenshot/apps/components/Button.js

```diff
@@ -256,6 +258,90 @@ const ButtonTests = [
 		<Button icon="arrowhookright" iconFlip="vertical">click me</Button>,
 		<Button icon="arrowhookright" iconFlip="both">click me</Button>,
 		<Button icon="arrowhookright" iconFlip="auto">click me</Button>
+	]),
+
+	// *************************************************************
+	// With customized button
+	// *************************************************************
+	// Note: When the file name of the test is too long, different tests may be recognized as the same test.
+	// So we changed the Button name of Focus prop tests.
+	...withProps({css: css}, [
```

Comment:
I think to test that exposed classes can be customized, It may be fine to reduce the number of cases.

Reply:
Suggested chagne.